requirements.txt

```diff
@@ -1,2 +1,2 @@
 click>=7.1.2
-papermill==2.2.0
+papermill==2.3.3
```

Comment:
should be safe to remove the version lock
can use `>=2.3.3` instead


unless you encountered a bug with later versions of `papermill`

Reply:
I've been bitten multiple times by supposedly-compatible updates on other projects so lock versions by default, but will make this update (and the others).